src/PetProject.Framework.Kafka/Consumer/Consumer.cs

```diff
@@ -91,65 +88,73 @@ public void Dispose()
         /// <summary>
         /// Decorator around Confluent Consumer to commit messages asynchronously after success.
         /// </summary>
-        public void CommitAsync()
+        public async Task<CommittedOffsets> CommitAsync()
         {
-            this.confluentConsumer.CommitAsync();
+            return await this.confluentConsumer.CommitAsync();
```

Comment:
you don't need to await the Task here, you can just return it

Reply:
ok, I'll remove the await.